src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -422,6 +418,36 @@ Actors:
         Collection: *coll
         IndexName: *metaAscTimeAsc
 
+- Name: Quiesce
```

Comment:
I think this would be better as the following the phases:

```
# Parameterized Quiesce Actor
QuiesceActor:
  Name: {^Parameter: {Name: "Name", Default: "FindOne"}}
  Type: QuiesceActor
  Threads: 1
  Database: {^Parameter: {Name: "Database", Default: *db_name}}
  Phases:
    OnlyActiveInPhases:
      Active: {^Parameter: {Name: "Active", Default: [0, 2, 4, 6, 8, 10]}}
      NopInPhasesUpTo: {^Parameter: {Name: "MaxPhases", Default: 20}}
      PhaseConfig:
        SleepBefore: 1 seconds
        SleepAfter: 1 seconds
        Repeat: 1
```

And then here would look like:

```
- LoadConfig:
    Path: *path
    Key: QuiesceActor
    Parameters:
      Active: [2, 5, 8, 11, 14, 17, 18, 21]
      MaxPhases: 21 # or whatever this is.
```


**Also why are there 2 consecutive quiesces?**

Reply:
Implemented the proposed changes, this significantly improves readability. 

Regarding the two consecutive quiesces, this does seem weird but is also the case in the original workload:

https://github.com/mongodb/genny/blob/4c87a4f5f743fd758b0786bf0e7d0484c2853063/src/workloads/query/TimeSeriesLastpoint.yml#L377
https://github.com/mongodb/genny/blob/4c87a4f5f743fd758b0786bf0e7d0484c2853063/src/workloads/query/TimeSeriesLastpoint.yml#L341

Should we reach out to the test authors about this or just keep as is?